internals/prod-scripts.js

```diff
@@ -0,0 +1,4 @@
+const execSync = require('child_process').execSync
+const { buildEnv } = require('./set-parcel-build-variables.js')
+
+execSync('yarn parcel:build', { env: buildEnv, stdio: 'inherit' })
```

Comment:
One simpler, but not cross-platform solution would be:
```json
"build": "REACT_APP_META_CSP=`yarn --silent print-csp` REACT_APP_BUILD_VERSION=`git rev-parse HEAD` REACT_APP_BUILD_TIME=`date +%s` yarn parcel:build"
```

Reply:
It's not working on my OS. I would keep cross-platform support and it's a bit easier to maintain